Docker-Swarm-deployment/single-node/resource-server/rs-stack.yaml

```diff
@@ -7,21 +7,19 @@ networks:
 
 services:
   rs:
-    image: ghcr.io/datakaveri/rs-depl:3.0-03915b1
+    image: iudx/rs-depl:latest
```

Comment:
please add latest 3.5 tagged image ghcr.io/datakaveri/rs-depl:3.5.0-e45e327. for all vertx servers. https://github.com/orgs/datakaveri/packages/container/package

Reply:
@abhi4578 
added latest 3.5 tagged image